diff --git a/vm/vmimpl/util.go b/vm/vmimpl/util.go
index 1146d47fd..8b8d8cb67 100644
--- a/vm/vmimpl/util.go
+++ b/vm/vmimpl/util.go
@@ -77,7 +77,7 @@ func SSHArgsForward(debug bool, sshKey string, port, forwardPort int, systemSSHC
 }

 func scpArgs(debug bool, sshKey string, port int, systemSSHCfg bool) []string {
-	return sshArgs(debug, sshKey, "-P", port, 0, systemSSHCfg)
+	return append([]string{"-O"}, sshArgs(debug, sshKey, "-P", port, 0, systemSSHCfg)...)
 }

 func sshArgs(debug bool, sshKey, portArg string, port, forwardPort int, systemSSHCfg bool) []string {
